developer_docs/RELEASE.md

```diff
@@ -25,30 +36,36 @@ SHELL := /bin/bash
 export VERSION ?= v1.0.0
 ```
 
-Say we want to perform a minor version release (i.e. no breaking changes and only new features and bug fixes are being included); in which case we'll update the _MINOR_ part of the version:
+Say we want to perform a minor version release (i.e. no breaking changes and only new features and bug fixes are being included); in which case we'll update the _MINOR_ part of the version, this can be done with the `make minor` target, but it should have been updated automatically via GitHub actions.
 
 ```Makefile
 SHELL := /bin/bash
 export VERSION ?= v1.1.0
 ```
 
+If a patch version needs to be released, the release will be done from the minor branch. For example, if we want to release `v1.5.1`, we will check out the `1.5` branch and perform any changes in that branch. The VERSION variable in the Makefile should already be up to date, but in case it's not, it can be bumped with the `make patch` target.
```

Comment:
you're using the term `patch` here when above in the semversion definition you have `PATCH` (excerpt: `in the SEMVER format: MAJOR.MINOR.BUG`) - can we make consistent?

Reply:
you're right, I'll fix that.